### ISSUE TITLE
ProcessSDAMessage error

### ISSUE THREAD

**Issue by Daeinar** [dedis/cothority#172]
I keep getting the following error:
```
0: (             sda.(*Host).ProcessMessages: 275) - ProcessSDAMessage returned: Not dispatching yet.
```
It seems it comes from: https://github.com/dedis/cothority/blob/sda_1/lib/sda/host.go#L549
Despite the error, sending messages through SDA back and forth seems to work. One can reconstruct it by running `go test` in `app/randhound` of the `pony_randhound` branch (which requires the `master` branch from `dedis/crypto` and the `interface` branch from `dedis/protobuf`).

**Comment by Liamsi** [comment]
You see this, because `DispatchToInstance` is `Still aggregating for this SDAData`. I'll try to find out howto wait for the msgs after lunch

**Comment by Liamsi** [comment]
I'm not sure why we return a boolean which indicates 'no we are not done yet' *and* an `error` which says the same. You can get rid of the error message by applying the patch
[err_msg.patch.txt](https://github.com/dedis/cothority/files/97666/err_msg.patch.txt)
 attached. (Sorry for the late reply, we were busy with some urgent homomorphic encryption)

**Comment by ineiti** [comment]
Is already fixed in #166 - nicolas, if you think it's OK to merge, go for it...

**Comment by Liamsi** [comment]
Ah great, just merge `sda_1`when #166 is merged.

**Comment by Liamsi** [comment]
> Is already fixed in #166 - nicolas, if you think it's OK to merge, go for it...
Can be closed.
